suite/tests/CMakeLists.txt

```diff
@@ -3972,10 +3972,11 @@ if (UNIX)
   endif ()
   # i#784: test app behavior on alarm
   tobuild(linux.alarm linux/alarm.c)
-  # XXX i#2043: enable for A64 once append_fcache_enter_prologue() is finished.
-  if (NOT APPLE AND NOT ANDROID AND NOT AARCH64) # Test uses Linux-specific timer code.
-    tobuild(linux.signal_race linux/signal_race.c)
-    target_link_libraries(linux.signal_race rt)
+  if (NOT APPLE AND NOT ANDROID) # Test uses Linux-specific timer code.
+    if (NOT AARCH64) # TODO i#2043: Enable this for AArch64.
```

Comment:
This PR fixes #2043, right?

Reply:
Yup, changed the comment to point to #1569. Will mark #2043 as fixed when creating the final commit message for this PR.